src_erl/NerlMonitor/src/MonitorGUI.py

```diff
@@ -0,0 +1,305 @@
+from term import Atom
+from pyrlang.node import Node
```

Comment:
@GuyPerets106 

pyrlang should be cloned as a submodule under the directory **src_py**

Then you use the following lines before importing pyrlang:

import sys
sys.path
sys.path.append('/usr/local/lib/nerlnet-lib/NErlNet/src_py')

from pyrlang.node import Node
...
...

Reply:
@GuyPerets106  
It's not relevant if you use virtualenv with pip -e install